app/helpers/uploads_helper.rb

```diff
@@ -9,7 +9,8 @@ class Postprocessor
     public
     def self.create_handler(name)
       name = name.to_s
-      PROCS[name] = lambda 
+      # from http://blog.gsamokovarov.com/ruby-tips-tricks-1, this works despite the warning
```

Comment:
No -- that creates a Proc, not a Lambda, both of which are distinct from Blocks, because thanks Ruby.  The difference being whether you can use `return` or not in it...  The idea was to define something that felt and could be programmed like a Method, but didn't violate your concern about actually allowing user-controlled text to trigger evaluating a method.  Methods can use return.  Blocks cannot.  Procs cannot.  Lambdas can.